tempus_fugit_minecraft/sound.py

```diff
@@ -4,21 +4,46 @@
 #issue 17
 class Sound:
     def __init__(self, file_path=None):
-        '''!Initializes the class. If a file path is named, uses that to load the sound file
-            @param string The file path for the sound file'''
+        """!
+        @brief 
+        Initializes the Sound class.
+        
+        @details If a file path is named, uses that to load the sound 
+        file
+        
+        @param file_path string The file path for the sound file
+        
+        @return None
```

Comment:
The return should be the initialization of the class.

Reply:
Both issues fixed. 
Thank you